SendToLogit/action.yml

```diff
@@ -0,0 +1,23 @@
+name: Send Log Event to Logit.io
+description: Send log record to logit.io  
+inputs:
+  LOGIT-API-KEY:
+    description: 'The Logit.io API key' 
+    required:    true
+  JSON:
+    description: 'Payload in Json Format'
+    required:    true
+runs:
+  using: composite
+  steps:
+       - name: Send to Logit.io 
+         shell: bash
+         id: send
+         run: |
+            curl -i  \
+                 -H "ApiKey: {{input.LOGIT-API-KEY}}"  \
+                 -H "Content-Type: application/json"  \
+                 -H "LogType: default"  \
```

Comment:
Maybe `"LogType: json"` instead ?
This is in step 2 here: https://logit.io/sources/configure/json
and here: https://logit.io/sources/configure/http